creation/create_cvmfsexec_distros.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+usage() {
+	echo "This script is used to generate cvmfsexec distributions for all"
+	echo "supported machine types (platform- and architecture-based)."
+	echo "The script takes one parameter {osg|egi|default} which specifies"
+	echo "the source to download the latest cvmfs configuration and repositories."
+}
+
+start=`date +%s`
+
+CVMFS_SOURCES=osg:egi:default
+# rhel6-x86_64 is not included; currently not supported due to EOL
+# egi for rhel8-x86_64 results in an error - egi does not yet have a centos8 build (as confirmed with Dave)
+# TODO: verify the logic when egi provides a centos8 build
+SUPPORTED_TYPES=rhel7-x86_64:rhel8-x86_64:suse15-x86_64
+cvmfsexec_temp=/tmp/cvmfsexec_pkg
```

Comment:
I'd use $TMPDIR in place of the hardcoded /tmp

Reply:
I'd use `mktemp -d -t  PREFIX`

Reply:
this has been updated (see latest commits)